### PR TITLE
[12.0][MIG] mrp_sale_info: Migration to 12.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
@Tecnativa TT23606

### CODE REVIEW

File: mrp_sale_info/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Antiun Ingenieria S.L. - Javier Iniesta
```
Comment (chienandalu) [inline mrp_sale_info/__openerp__.py]: Should be `__manifest__.py` isn't it?

**Review by chienandalu — COMMENTED** [review]
Tested. Minor remark:

File: mrp_sale_info/__openerp__.py (hunk 1 above)
Comment (pedrobaeza) [reply mrp_sale_info/__openerp__.py]: Yes, something happened with the renaming commit...

Comment (pedrobaeza) [reply mrp_sale_info/__openerp__.py]: Changed

**Comment by pedrobaeza** [comment]
@CasVissers-360ERP @sergiocorato do you agree with this version?

**Review by sergiocorato — APPROVED** [review]
LGTM

**Review by CasVissers-360ERP — APPROVED** [review]

**Comment by CasVissers-360ERP** [comment]
Technical and functional review in: 
http://3434917-521-abf1a2.runbot2-3.odoo-community.org/web?debug=1#id=4&action=328&model=mrp.production&view_type=form&menu_id=176

Nice additions would be a smartbutton on SO and MO.

**Comment by pedrobaeza** [comment]
Great, let's merge it and we can add the suggestions as Roadmap:

/ocabot merge nobump

**Pull request merged**